fixtures/fixture.only.spec.yaml

```diff
@@ -0,0 +1,26 @@
+apiCalls:
+    swagger:
+      - name: Login - success
+        only: true
```

Comment:
@dreich-nucleus The exclusivity feature allows you to run only the specified test-case by appending. 

It is useful if you need to debug only one (or a few tests) among others in the same test suite.

 It works the same way the mocha `only` works checkout (https://mochajs.org/#exclusive-tests). 

Yes, it works as an opposite of `skip.`, Instead of skipping the test it will skip all the others and execute only the one marked as only: true